deps/spidershim/test/v8v8.cc

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "v8engine.h"
+
+#include "gtest/gtest.h"
+
+TEST(SpiderShim, Version) {
+  v8::V8::GetVersion();
```

Comment:
Seems to be [stable](https://github.com/mozilla/spidernode/blob/a01a274ef82b589fc846b686e1f243fc47b92df3/deps/spidershim/spidermonkey/js/src/jsapi.cpp#L712) now.  We can change the test later if this changes in the future...